packages/ckeditor5-engine/src/model/typecheckable.ts

```diff
@@ -22,20 +22,80 @@ import type Text from './text';
 import type TextProxy from './textproxy';
 
 export default abstract class TypeCheckable {
+	/**
+	 * Checks whether the object is of type {@link module:engine/model/node~Node} or its subclass.
+	 */
```

Comment:
Maybe it would be good to add some examples as it previously was documented in each individual class?